apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx

```diff
@@ -173,10 +171,10 @@ export function OrderRow({
   }, [orderActions, order])
   const alternativeOrderModalContext = useMemo(
     () => orderActions.getAlternativeOrderModalContext(order),
-    [order, orderActions]
+    [order, orderActions],
   )
 
-  const withAllowanceWarning = hasEnoughAllowance === false && hasValidPendingPermit === false
```

Comment:
Does this mean we no longer consider a pending permit for the allowance message?

Reply:
https://github.com/cowprotocol/cowswap/pull/5207#discussion_r1886795296